packages/@react-aria/searchfield/test/useSearchField.test.js

```diff
@@ -67,12 +67,10 @@ describe('useSearchField hook', () => {
         onSubmit.mockClear();
       });
 
-      it('preventDefault is called for Enter and not Escape', () => {
+      it('preventDefault is not called for Escape', () => {
```

Comment:
Can we combine this test and the following?
```suggestion
      it('preventDefault and stopPropagation are not called for Escape', () => {
```

Reply:
I've merged the two test cases.